.github/workflows/build.yml

```diff
@@ -100,7 +100,7 @@ jobs:
               run: |-
                 sudo dpkg --add-architecture i386
                 sudo apt-get update
-                sudo apt install --yes --no-install-recommends gcc-multilib g++-multilib libc6-dev-i386 pkg-config-i686-linux-gnu libwayland-dev:i386 libxrandr-dev:i386
+                sudo apt install --yes --no-install-recommends gcc-multilib g++-multilib libc6-dev-i386 pkg-config-i686-linux-gnu libwayland-dev:i386 libxrandr-dev:i386 libx11-dev:i386
```

Comment:
Could this be the 'cause' of linux 32 bit CI failing? or is it more likely to be missing other packages as well. The CI log didn't make it clear to me which was which.

Reply:
Wait, seems that line 114 of CMakeLists corresponds to
```
find_package(DirectFB REQUIRED)
```
Which... is surprising to me. BUILD_WSI_DIRECTFB_SUPPORT is off, so why is it trying to find that package?

Reply:
IGNORE THAT.
I was looking at `main` and not your branch. So of COURSE the line numbers didn't match up.

Reply:
The actual line appears to be `pkg_check_modules(XCB REQUIRED QUIET IMPORTED_TARGET xcb)` which is much more in line with the code changes.